client/src/components/Dog/DogCard.js

```diff
@@ -38,6 +39,7 @@ const DogCard = ({ dog }) => {
   const dogImage = image ? image : "placeholder-dog.svg";
 
   return (
+    <Link className="dogInfoLink" to={`/dog/${dog._id}`} style={{textDecoration: 'none'}}>
```

Comment:
Since we already have `useStyles()` here, instead of inline styling (which is generally not recommended), you can do 

```suggestion
    <Link className={classes.dogInfoLink} to={`/dog/${dog._id}`} >
```
and define the styles for dogInfoLink there

Reply:
amended in latest commit

Reply:
Missing the styles for `dogInfoLink`, but after this should be ok for now!:

![Screenshot 2021-03-31 at 6 53 47 PM](https://user-images.githubusercontent.com/26196990/113133645-7900c800-9252-11eb-9a89-b212618145e0.png)

Will there be a separate story card to work on jazzing up the UI?

Reply:
Yeah a separate story card, so we can focus on making sure it works first, then spruce it up later